returnn/hdf.py

```diff
@@ -309,12 +310,25 @@ def __init__(
         self.allophone_file = allophone_file
         self.state_tying_file = state_tying_file
         self.out_hdf_files = [self.output_path(f"data.hdf.{d}", cached=False) for d in range(len(alignment_caches))]
+        self.out_excluded_segments = self.output_path(f"excluded.segments", cached=False)
```

Comment:
PR looks good. Wondering `cached` needs to be set, since `False` the default I think and we can assume this does not change? 
But this does not hinder approval.